packages/button/src/button.tsx

```diff
@@ -138,6 +143,19 @@ export const Button = forwardRef<ButtonProps, "button">((props, ref) => {
       {isLoading
         ? loadingText || <chakra.span opacity={0}>{children}</chakra.span>
         : children}
+      {isLoading && loadingSpinnerPosition === "end" && (
+        <ButtonSpinner
+          __css={{
+            fontSize: "1em",
+            lineHeight: "normal",
+            marginLeft: iconSpacing,
+          }}
+          spacing={iconSpacing}
```

Comment:
This seems wrong. Consider handling the `iconSpacing` within `ButtonSpinner`.

```jsx
// Handle the placement within the ButtonSpinner
<ButtonSpinner placement="end" spacing={iconSpacing} />
```

- Use `marginStart` instead of `marginLeft` to get it working in RTL layouts

Reply:
See [Right To Left](https://developer.mozilla.org/en-US/docs/Web/CSS/direction)
[Chakra UI supports ltr and rtl out of the box](https://chakra-ui.com/docs/features/rtl-support), so we need so keep an eye on this :)